addons/binding/org.openhab.binding.exec/ESH-INF/i18n/exec_de.properties

```diff
@@ -0,0 +1,33 @@
+###############
+# binding
+binding.exec.name = Exec Binding
+binding.exec.description = Binding zur Ausf�hrung von Befehlen und zur Verarbeitung des R�ckgabewerts
+
+###############
+# thing types
+thing-type.exec.command.label = Befehl
+thing-type.exec.command.description = Verarbeitung des R�ckgabewertes bei Ausf�hrung des Befehls
+
+# thing type configuration
+thing-type.config.exec.command.command.label = Befehl
+thing-type.config.exec.command.command.description = Der auszuf�hrende Befehl
+thing-type.config.exec.command.transform.label = Transformation
+thing-type.config.exec.command.transform.description = Transformation des R�ckgabewerts des Befehls, z.B. REGEX((.*)) liefert den R�ckgabewert unver�ndert
+thing-type.config.exec.command.interval.label = Intervall
+thing-type.config.exec.command.interval.description = Intervall in Sekunden in welchem der Befehl ausgef�hrt wird
+thing-type.config.exec.command.timeout.label = Timeout
+thing-type.config.exec.command.timeout.description = Timeout in Sekunden nachdem die Ausf�hrung des Befehls abgebrochen wird
```

Comment:
Timeout in Sekunden, nach dem ...

Reply:
@bjoernbrings Can you check this? Thanks.